backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -72,7 +73,7 @@ public IActionResult GetDocumentTypes()
         //[HasPermission(Permissions.NoteView)]
         [ProducesResponseType(typeof(IList<DocumentRelationshipModel>), 200)]
         [SwaggerOperation(Tags = new[] { "document" })]
-        public IActionResult GetActivityDocuments(DocumentRelationType relationshipType, long parentId)
+        public IActionResult GetRelationshipDocuments(DocumentRelationType relationshipType, long parentId)
```

Comment:
We are still missing relevant permissions here? Did you file a JIRA to address that?

Reply:
I dont think there is a jira for documents permissiosn just yet

Reply:
would be nice if you file one as you are the developer for this feature - I worry we forget about permissions. Usually permissions are set on the story that handles update/read/create like this one (and the one before). 

If it wasn't called out specifically then we could end up not having permissions in place for this feature

Reply:
more of a process thing - perhaps we can discuss process re: when to add claims to DB on Friday